collectors/ebpf.plugin/README.md

```diff
@@ -8,28 +8,24 @@ sidebar_label: "eBPF"
 
 # eBPF monitoring with Netdata
 
-eBPF consists of a wide toolchain that ultimately outputs a set of bytecode that will run inside the eBPF virtual
-machine (VM) which lives inside the Linux kernel. The program in particular is executed in response to a [tracepoint
-or kprobe](#probes-and-tracepoints) activation.
+The Netdata Agent provides many [eBPF](https://ebpf.io/what-is-ebpf/) programs to help you troubleshoot and debug how applications that interact with the Linux kernel. The `ebpf.plugin` uses [tracepoints or kprobes](#data-collection) to collect a wide array of high value data about the host that would otherwise be impossible to capture. 
```

Comment:
@kickoke, please, update the next line from 

`The `ebpf.plugin` uses [tracepoints or kprobes](#data-collection)`

to

The `ebpf.plugin` uses [tracepoints, trampoline, and kprobes](#data-collection)